bedrock/pocket/views.py

```diff
@@ -13,3 +20,28 @@ def server_error_view(request, template_name="pocket/500.html"):
 def page_not_found_view(request, exception=None, template_name="pocket/404.html"):
     """404 error handler that runs context processors."""
     return l10n_utils.render(request, template_name, ftl_files=["pocket/404"], status=404)
+
+
+@require_POST
+def newsletter_subscribe(request):
+    try:
+        data = json.loads(request.body)
+    except Exception:
+        return JsonResponse({"error": "Error parsing JSON data"}, status_code=400)
+
+    external_id = request.COOKIES.get("a95b4b6") or None
```

Comment:
@pmac is this a specific ID for all Pocket-related stuff or will it change in prod?

Reply:
This is the cookie ID that the pocket folks gave me. I think it's consistent.